src/binary.ts

```diff
@@ -31,6 +31,10 @@ export class Binary {
   get _bsontype(): 'Binary' {
     return 'Binary';
   }
+  /** @internal */
```

Comment:
Will all types always have the same version or do we expect in the future each type could have a different version for, say, the minimum serializer version it matches? The reason I ask is that if every type will always have the same version could we dry this up so it's only defined in one place and all the types use that?